Build fails for latest v11.0.0

I have been using `jitsi_meet_flutter_sdk: ^10.3.0` for a while now and all good so far, but after updating to latest v11.0.0 I have the following issues:

## iOS
```bash
Could not build the precompiled application for the device.
Swift Compiler Error (Xcode): No such module 'JitsiMeetSDK'
/Users/user/.pub-cache/hosted/pub.dev/jitsi_meet_flutter_sdk-11.0.0/ios/Classes/JitsiMeetPlugin.swift:2:7
```

## Android
```bash
e: file:///Users/user/.pub-cache/hosted/pub.dev/jitsi_meet_flutter_sdk-11.0.0/android/src/main/kotlin/org/jitsi/jitsi_meet_flutter_sdk/WrapperJitsiMeetActivity.kt:106:41 Unresolved reference 'CUSTOM_OVERFLOW_MENU_BUTTON_PRESSED'.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jitsi_meet_flutter_sdk:compileReleaseKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
```

------------------------
I'm using `ios, '15.5` as minimum for iOS and `minSdkVersion 26` as minimum for Android.
This is my `flutter doctor -v` output:

```bash
[✓] Flutter (Channel stable, 3.29.0, on macOS 15.3 24D60 darwin-arm64, locale en-US) [642ms]
    • Flutter version 3.29.0 on channel stable at /Users/user/fvm/versions/3.29.0
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 35c388afb5 (9 days ago), 2025-02-10 12:48:41 -0800
    • Engine revision f73bfc4522
    • Dart version 3.7.0
    • DevTools version 2.42.2

[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0) [3.5s]
    • Android SDK at /Users/user/Android/SDK
    • Platform android-35, build-tools 35.0.0
    • ANDROID_HOME = /Users/user/Android/SDK
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
      This is the JDK bundled with the latest Android Studio installation on this machine.
      To manually set the JDK path, use: `flutter config --jdk-dir="path/to/jdk"`.
    • Java version OpenJDK Runtime Environment (build 21.0.5+-12932927-b750.29)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 16.2) [2.4s]
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 16C5032a
    • CocoaPods version 1.16.2

[✓] Chrome - develop for the web [164ms]
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2024.2) [164ms]
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = /Applications/Android Studio.app/
    • Java version OpenJDK Runtime Environment (build 21.0.5+-12932927-b750.29)
```

--- a/android/src/main/kotlin/org/jitsi/jitsi_meet_flutter_sdk/WrapperJitsiMeetActivity.kt
+++ b/android/src/main/kotlin/org/jitsi/jitsi_meet_flutter_sdk/WrapperJitsiMeetActivity.kt
@@ -103,7 +103,7 @@ class WrapperJitsiMeetActivity : JitsiMeetActivity() {
 
                     BroadcastEvent.Type.READY_TO_CLOSE.action -> eventStreamHandler.readyToClose()
 
-                    BroadcastEvent.Type.CUSTOM_OVERFLOW_MENU_BUTTON_PRESSED.action -> eventStreamHandler.customOverflowMenuButtonPressed(
+                    BroadcastEvent.Type.CUSTOM_BUTTON_PRESSED.action -> eventStreamHandler.customOverflowMenuButtonPressed(
                         data
                     )
 

